angular-workspace/projects/ni/nimble-angular/src/directives/anchor-button/tests/nimble-anchor-button-router-link.directive.spec.ts

```diff
@@ -22,10 +22,8 @@ describe('Nimble anchor button RouterLinkDirective', () => {
             declarations: [TestHostComponent],
             imports: [NimbleAnchorButtonModule,
                 CommonModule,
-                RouterTestingModule.withRoutes(
-                    [{ path: '', component: TestHostComponent, pathMatch: 'full' }
-                    ], { useHash: true }
-                )
+                RouterTestingModule.withRoutes([{ path: '', component: TestHostComponent, pathMatch: 'full' }
```

Comment:
It's odd that it didn't chose to put both parameters on line or both on new lines like in the next file, and instead chose to put the closing bracket with the second parameter on a new line. This is probably because it was originally that way.

Reply:
Good point, I think you're right. I tidied it up manually and re-ran the formatter and it didn't modify it back.